### PR TITLE
ADQL editor fixes when service url changes

### PR DESCRIPTION
**Pull request by brianv0**
Fixes issues with adql text area being out of sync when service url changes as well as z-index related issues when selecting a new service url

### CODE REVIEW

File: src/firefly/js/ui/tap/TableSelectViewPanel.jsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ export class TapSearchPanel extends PureComponent {
 
                         <div className='TapSearch__section'>
                             <div className='TapSearch__section--title'>1. TAP Service <HelpIcon helpId={tapHelpId('tapService')}/> </div>
-                            <div style={{flexGrow: 1, marginRight: 3, maxWidth: 1000}}>
+                            <div style={{flexGrow: 1, marginRight: 3, maxWidth: 1000, zIndex: 9999}}>
```
Comment (loitly) [inline src/firefly/js/ui/tap/TableSelectViewPanel.jsx:187]: Can you describe when this is needed?  I am looking at `dev` now, and it looks like it's working fine without the change.  With this change is when the Schema and Table input fields flow on top of the TAP Service dropdown.

Comment (brianv0) [reply src/firefly/js/ui/tap/TableSelectViewPanel.jsx:187]: image

See image ^. In the case when you are selecting the bottom of GAIA or the top of GAVO, the textarea is actually z-index=1 and it is above those divs (TapSearch__section) and selection won't actually work in that limited area, but it works in every other area of the dropdown.

Comment (loitly) [reply src/firefly/js/ui/tap/TableSelectViewPanel.jsx:187]: ah.  thanks.  i was looking at the wrong section.

**Review by loitly — APPROVED** [review]
Looks fine.  Good to go.

**Review by robyww — APPROVED** [review]
Looks Good.

**Pull request merged**